<ui version="4.0" >
 <class>ICQHomeInfoWidget</class>
 <widget class="QWidget" name="ICQHomeInfoWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>283</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Location &amp;&amp; Contact Information</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>&amp;Address:</string>
        </property>
        <property name="buddy" >
         <cstring>addressEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QLineEdit" name="addressEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>&amp;City:</string>
        </property>
        <property name="buddy" >
         <cstring>cityEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="cityEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="textLabel4" >
        <property name="text" >
         <string>&amp;State:</string>
        </property>
        <property name="buddy" >
         <cstring>stateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="stateEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>&amp;Zip:</string>
        </property>
        <property name="buddy" >
         <cstring>zipEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="zipEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="textLabel8" >
        <property name="text" >
         <string>Countr&amp;y</string>
        </property>
        <property name="buddy" >
         <cstring>countryCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QComboBox" name="countryCombo" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="textLabel5" >
        <property name="text" >
         <string>&amp;Phone:</string>
        </property>
        <property name="buddy" >
         <cstring>phoneEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="phoneEdit" >
        <property name="text" >
         <string/>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="textLabel6_2" >
        <property name="text" >
         <string>Ce&amp;ll:</string>
        </property>
        <property name="buddy" >
         <cstring>cellEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLineEdit" name="cellEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="textLabel7_2" >
        <property name="text" >
         <string>Fa&amp;x:</string>
        </property>
        <property name="buddy" >
         <cstring>faxEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="faxEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="textLabel9" >
        <property name="text" >
         <string>&amp;Email:</string>
        </property>
        <property name="buddy" >
         <cstring>emailEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3" >
       <widget class="QLineEdit" name="emailEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="textLabel10_2" >
        <property name="text" >
         <string>&amp;Homepage:</string>
        </property>
        <property name="buddy" >
         <cstring>homepageEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3" >
       <widget class="QLineEdit" name="homepageEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>466</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>addressEdit</tabstop>
  <tabstop>cityEdit</tabstop>
  <tabstop>stateEdit</tabstop>
  <tabstop>zipEdit</tabstop>
  <tabstop>countryCombo</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>cellEdit</tabstop>
  <tabstop>faxEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>homepageEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
